tools/worker/work_processor.cc

```diff
@@ -143,6 +174,43 @@ void WorkProcessor::ProcessWorkRequest(
     }
   }
 
+  // Import global indexes if compilation succeeds
+  if (exit_code != EXIT_FAILURE && enable_global_index_store) {
+    std::vector<std::string> ii_args;
+
+    // The index-import runfile path is pased as a define
+    #if defined(INDEX_IMPORT_PATH)
+      ii_args.push_back(INDEX_IMPORT_PATH);
+    #else
+      // Logical error
+      std::cerr << "Incorrectly compiled work_processor.cc";
+      exit_code = EXIT_FAILURE;
+    #endif
+
+    // Use the output path map to deterine what compilation ouputs to import
+    auto outputs  = output_file_map.incremental_outputs();
+    std::map<std::string, std::string>::iterator it;
+    for (it = outputs.begin(); it != outputs.end(); it++) {
+      // Need the actual output paths of the compiler - not bazel
+      auto output_path =  is_wmo ? it->first : it->second;
+
+      auto file_type = output_path.substr(output_path.find_last_of(".") + 1);
+      if (file_type == "o") {
+        ii_args.push_back("-import-output-file");
+        ii_args.push_back(output_path);
+      }
+    }
+
+    // Copy back from the global index store to bazel's index store
+    ii_args.push_back(global_index_store);
+
+    ii_args.push_back(exec_root + "/" + bazel_index_store_path);
+
+    const std::vector<std::string>& ii_args_ = ii_args;
+    // Dupes the output into the action for failure cases
```

Comment:
```suggestion
    // Dupes the output into the action for failure cases
    const std::vector<std::string>& ii_args_ = ii_args;
```

Why do we need to do this assignment?

Reply:
I think the function is looking for a `const`